data/json/items/melee/spears_and_polearms.json

```diff
@@ -1035,5 +1035,18 @@
     "price_postapoc": "30 USD",
     "material": [ "wood", "qt_steel" ],
     "melee_damage": { "bash": 18, "cut": 26 }
+  },
+  {
+    "type": "GENERIC",
+    "id": "trianja",
+    "name": { "str": "trianja" },
+    "description": "When the Cataclysm struck, some worlds fought back as best they could, no matter their level of technological progress.  This strangely shaped pike resembles a trident with pronounced outer prongs, wide enough to pin a zombie to the ground from afar as well as skewer them with the center spike.  They didn't stop armageddon, but were effective enough for the Exodii to consider bringing along, if not to use then to sell.",
+    "copy-from": "pike_bronze",
+    "price_postapoc": "60 USD",
+    "volume": "3850 ml",
+    "weight": "2650 g",
+    "melee_damage": { "cut": 20, "stab": 31 },
+    "weapon_category": [ "POLEARMS", "HOOKING_WEAPONRY" ],
+    "techniques": [ "WBLOCK_1", "PENETRATE", "PIN" ]
```

Comment:
You didn't address any of my points about this, it's a non-weapon, not meaningfully different from any other repurposed pitchfork or fishing trident, i.e. terrible.
```suggestion
```